lib/dice_bag/available_templates.rb

```diff
@@ -16,6 +16,23 @@ def templates_location
 
     class << self
 
+      def gem_specs
+        @gem_specs ||= begin
+          Gem::Specification.sort.each_with_object({}) do |spec, hsh|
+            hsh[spec.name] = spec.full_gem_path
+          end
+        end
+      end
+
+      def checker_within_given_gems?(checker, gem_names)
+        return true if gem_names.empty?
```

Comment:
but then it would generate no template at all when the `all` method is called (by the regular generate_all cmd etc).
I'll try to think about alternatives but it might be very similar in the end.